app/views/posts/_post_details.html.erb

```diff
@@ -2,14 +2,36 @@
 <div class="post">
   <div class="comment-header">
     <h2>
-      Post by <%= post.author.name %>
+      Post by <%= post.user.name %>
     </h2>
     <small>
-      <span>Comment: <%= post.comments_counter %>, Likes: <%= post.likes_counter %></span>
+      <span>
+      <%= render partial: "likes/form", locals: { like: Like.new, user: @post.user, post: @post } %>
```

Comment:
- [ ] [REQUIRED] Kindly ensure a user is enabled to like a post because currently, I can not like the post created.

![Screenshot from 2022-06-10 09-38-14](https://user-images.githubusercontent.com/58692590/173027170-2b8f86dc-a0c2-4744-bcf2-990fa129052e.png)

Reply:
@Whoistolu can you please press the like counter text it will like the post. The text is clickable

Reply:
> @Whoistolu can you please press the like counter text it will like the post. The text is clickable

Hey, It worked. Apologies for that. For a better User Experience(UX) you can consider making it a button. 
NB: Next reviewer, please note tat the project is complete.

Reply:
But now i added the like button
which does the liking

Reply:
NB: Next reviewer, please note that the project is now complete.
![Screenshot from 2022-06-10 10-15-16](https://user-images.githubusercontent.com/58692590/173033510-e35af820-91b9-4bd7-a9c7-3c8c5569c10e.png)

Reply:
@Whoistolu cool 👍